### PR TITLE
updating module to support auto-import

### PR DESCRIPTION
**Pull request by dgraeber**
*Issue #, if available:*
#133 
*Description of changes:*
Update the module to accept auto-import configs via CDK for SCRATCH_2 storage types

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: 427e84f81e84f186451b3339540bc2a382f66b28
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=498a5d24-2dff-4e85-aede-7a6fea7ee32d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by srinivasreddych — DISMISSED** [review]

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: 459aa9e473c5f70251fad6a025ac3a4588274f18
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=2467825a-7ccf-4104-b726-66c48b10ec33%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: 7462f81641ae4c0903e34dd8087f8d6b49149d9c
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=40351e62-e5e9-4a19-95dd-60cd2800fd7d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,29 @@ All notable changes to this project will be documented in this file.
...
 
+
 =======
 
 ## UNRELEASED
 
 ### **Added**
 
+### **Changed**
+- modified `fsx-luster` module to auto-import data if configured
```
Comment (malachi-constant) [inline CHANGELOG.md:16]: little typo

**Review by malachi-constant — APPROVED** [review]
LGTM, just one typo in the changelog

**Review by kukushking — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dgraeber) [reply CHANGELOG.md:16]: On it...and need to add more context to the README

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: IDFModuleTestsInfrastructur-59JoFoVH64CI
* Commit ID: a82caf9588f307219cd7efca497e8b652feb2968
* Result: SUCCEEDED
* [Build Logs](https://mz6i2qapub.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=81abc205-0b30-43f3-9962-6324d5f550aa%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by malachi-constant — APPROVED** [review]

**Pull request merged**